src/app/shared/components/person-card/person-card.component.html

```diff
@@ -1,33 +1,59 @@
 <div>
-  <a [routerLink]="" class="edit-icon"><button mat-button style="color:#3849F9;">
-      <mat-icon style="color:#3849F9;">edit</mat-icon>Редагувати
-    </button></a>
-  <ng-container *ngIf="isChildInfo then ChildInfo; else ParentInfo"></ng-container>
+  <mat-card>
+    <div *ngIf="(CardType.child === cardType) then ChildView; else ParentView"></div>
+  </mat-card>
+  <div *ngIf="(CardType.user === cardType) then UserView"></div>
 </div>
 
-<ng-template #ChildInfo>
-  <mat-card class="card">
-    <mat-card-content fxLayout='column' fxLayoutAlign='center start'>
-      <h4>ПІБ</h4>
-      <p>{{card.firstName}} {{card.lastName}} {{card.secondName}}</p>
-      <h4>День народження</h4>
-      <p>{{card.birthDay}}</p>
-      <h4>Cтать дитини</h4>
-      <p>{{card.gender}}</p>
-      <h4>Соціальна група</h4>
-      <p>Відсутня</p>
-    </mat-card-content>
-  </mat-card>
+
+<ng-template #ChildView>
+
+  <label class="card_label">ПІБ</label>
+  <p class="card_text">{{ card.lastName }} {{ card.firstName }} {{ card.middleName }} </p>
+  <label class="card_label">Дата народження</label>
+  <p class="card_text"> {{ card.dateOfBirth }} </p>
+  <label class="card_label">Стать дитини</label>
+  <p class="card_text"> {{ card.gender === 0 ? 'Чоловіча' : "Жіноча" }} </p>
+  <label class="card_label">Соціальна група</label>
+  <p class="card_text"> {{ card.socialGroupId === 0 ? 'Відстуня' : "Присутня" }} </p>
+
 </ng-template>
-<ng-template #ParentInfo>
-  <mat-card class="card">
-    <mat-card-content fxLayout='column' fxLayoutAlign='center start'>
-      <h4>ПІБ</h4>
-      <p>{{card.firstName}} {{card.lastName}} {{card.secondName}}</p>
-      <h4>Телефон</h4>
-      <p>{{card.phone}}</p>
-      <h4>Емейл</h4>
-      <p>{{card.email}}</p>
-    </mat-card-content>
-  </mat-card>
+
+<ng-template #ParentView>
+
+  <label class="card_label">ПІБ</label>
```

Comment:
when we started to use underscore in classnames? I believe we use dash in the project